GoogleMeetup/src/main/webapp/style.css

```diff
@@ -1,9 +1,353 @@
+    font-size: 16px;
+    font-weight: 400;
+    font-family: inherit;
+    margin: 0;
+    padding: 8px 20px; 
+    display: block;
+    color: inherit;
+    text-decoration: none;
+}
+
+#navbar .clm h3 {
+    font-size: 20px;
+    font-weight: 700;
+}
+
+#navbar .sub-item {
+    background-color: #FFFFFF;
+    position: relative;       
+    transition: all 0.3s;
+}
+
+#navbar .dropdown li:hover > span, #navbar .dropdown li:hover > a {
+    color: #000000;
+    background-color: #EEEEEE;
+}
+
+#navbar .clm a:hover {
+    color: #000000;
+    transition: color 0.2s;
+}
+
+#navbar .dropdown.right0 {
+    left: auto;
+    right: 0;
+} 
+
+#navbar .dropdown li > .dropdown.right0 {
+    left: auto;
+    right: 100%;
+} 
+       
+#navbar li.full-width {
+    position: static;
+}
+
+#navbar li.full-width .dropdown {
+    width: 100%;
+    left: 0;
+    box-sizing: border-box;
+}  
+
+#navbar li:hover > .dropdown {
+    display: block;
+    opacity: 1;
+    z-index: 1;
+}
+
+#navbar .dropdown li > .dropdown {
+    left: 100%; 
+    right: auto;
+    top: 0;
+}
+
+#navbar ul.dropdown {
+    min-width: 240px; 
+}
+
+#navbar div.dropdown  {
+    text-align: center;
+}
+
+/* each column */
+#navbar .clm {
+    text-align: left;
+    margin: 20px;
+    vertical-align: top; /* middle */
+    width: auto;
+    min-width: 240px;
+    display: inline-block;
+    *display: inline;
+    *zoom: 1;
+}
+
+#navbar .clm a:hover {
+    color: #000000;
+    text-decoration: underline;
+}
+
+/*-----------Arrows----------------*/
+#navbar .arrow {
+    color: inherit;
+    border-style: solid; 
+    border-width: 2px 2px 0 0; 
+    padding: 5px; 
+    transform: rotate(135deg);
+    margin-top: -10px;
+    margin-left: 5px;
+    position: relative;
+    display: inline-block;
+    width: 0;
+    height: 0;
+    vertical-align: middle;
+    overflow: hidden;
+}
+        
+#navbar .dropdown .arrow {
+    transform: rotate(45deg);
+    top: 50%;
+    margin-top: -7px;
+    position: absolute;
+    left: auto;
+    right: 20px;
+} 
+
+
+
+@keyframes topItemAnimation{
+    from {opacity: 0;  transform:translate3d(-16%, 0, 0);}
+    to {opacity: 1; transform:translate3d(0, 0, 0);}
+}
+#navbar li a {animation:none;}
+#navbar.active li a { animation: topItemAnimation 0.5s ease 0.5s backwards;}
+#navbar.active li:nth-of-type(1) a { animation: topItemAnimation 0.5s ease 0.08333333333333333s backwards;}
+#navbar.active li:nth-of-type(2) a { animation: topItemAnimation 0.5s ease 0.16666666666666666s backwards;}
+#navbar.active li:nth-of-type(3) a { animation: topItemAnimation 0.5s ease 0.25s backwards;}
+#navbar.active li:nth-of-type(4) a { animation: topItemAnimation 0.5s ease 0.3333333333333333s backwards;}
+#navbar.active li:nth-of-type(5) a { animation: topItemAnimation 0.5s ease 0.4166666666666667s backwards;}@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) { #navbar.active li a{animation:none!important;}}
+
+
+
+
+
+
```

Comment:
Given the way you've structured the file, it feels like these should be multiple CSS files.